caramel/scripts/tool.py

```diff
@@ -79,22 +79,35 @@ def error_out(message):
 
 
 def print_list():
```

Comment:
This formatting does not look right.  run `black`  on the code before commit. ( `pip install black` )